### ISSUE TITLE
1.20 port

### ISSUE THREAD

**Issue by EysseW** [coolsimulations/ServerStop#6]
Hi. I really really love this mod. My server depends on it. Is there any plans so far for a 1.20 port?

**Comment by coolsimulations** [comment]
Yes. I am currently working on the 1.20.1 update and a new version of the mod. This will take longer than usual to come out just because I'm rewriting the mod from scratch in a new setup that will let me work on both the Forge and Fabric versions simultaneously, so it's easier and quicker to update. Hopefully in this new version the shutdown warning messages are more reliable. I'm posting regular development updates in the discord server if you want to stay up to date with where I'm at.

**Comment by EysseW** [comment]
That's awesome! Thank you!
